### PR TITLE
Fix playback of purchased movies - Old issue #837

### PR DESCRIPTION
**Pull request by MisterD81**
This PR fixes the handling of purchased movies. See here: https://github.com/jdf76/plugin.video.youtube/issues/837

Therefor two changes were done:
- Use generate_mpd also when a license URL is available and add the ContentProtection section to MPD when the license URL is available
- Extract Languages from YT response and based on those languages build the AdaptationSet based also on the languages. For me the "Default" must be False as otherwise Kodi would select a wrong language.

The change looks bit, but I decided to change the variable name for the key in method "generate_mpd" as it now also could contain a language and that confused me sometimes.

### CODE REVIEW

**Comment by anxdpanic** [comment]
Thanks! This looks great, I'm assuming this has been well tested with regular videos as well?
Also, are you participating in Hacktoberfest? If so I'll add the topic before approving and merging, not sure if it had to be in place before the PR but hopefully it'd count for you.

**Comment by MisterD81** [comment]
I did some test with normal videos as well for sure, but I could not find easily a video that has multiple audio tracks which could be affected by this PR.
I'm not participating in the Hacktoberfest. I just use the plugin for watching my bought videos at Google Play which are also available in Youtube ;) It definitely increases WAF when she only needs to click "OK" in Kodi to play a movie.

**Comment by anxdpanic** [comment]
hah, absolutely! and thanks again, we'll cross the multi stream audio bridge when we come to it ;)

**Review by anxdpanic — APPROVED** [review]

**Pull request merged**
